<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="comboMode">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Open-loop frequency counter/phase meter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Closed-loop fiber noise canceler</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="editModulatorNominalFreq">
     <property name="text">
      <string>20</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Modulator nominal freq [MHz]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Modulator sign
(does AOM shifts freq up or down?)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblDetectedModulatorGain">
     <property name="text">
      <string>TODO!</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Detected modulator gain [unitless]</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="editModulatorGain">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Target closed-loop bandwidth</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboTargetBW"/>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QComboBox" name="comboModulatorSign">
     <item>
      <property name="text">
       <string>Upshift</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Downshift</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Modulator gain [unitless]
(nb of passes through AOM)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Channel mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Expected freq at input [MHz]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editExpectedFreq"/>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLock">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lock</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
